docs/resources/user_account_password.md

```diff
@@ -26,7 +26,7 @@ description: |-
 
 This Terraform resource is used to update password for a user of the iDRAC Server.
 
-~> **Note:** Only user with role as "Administrator" has privilige to update password using this resource. For other users, use user module or user resource.
+~> **Note:** This resource can be used by users with 'Administrator' role/privilege. For users with non-administrator role/privilege, either 'user resource' or 'user module' can be used for updating the password.
```

Comment:
can we give link to user module?

Reply:
we cannot give any relative links here, if there is any particular URL then we can give